frontend/src/constants/queryBuilder.ts

```diff
@@ -246,6 +246,14 @@ export enum PANEL_TYPES {
 	EMPTY_WIDGET = 'EMPTY_WIDGET',
 }
 
+// eslint-disable-next-line @typescript-eslint/naming-convention
+export enum DISPLAY_TYPES {
```

Comment:
i am not sure whether we need this enum or not

Reply:
Although the location is debatable, enums will make our display values more safe and clear since they'll be floating around in different parts of code.